src/applications/disability-benefits/2346/components/IntroductionPage.jsx

```diff
@@ -30,26 +29,33 @@ class IntroductionPage extends React.Component {
         <div className="process schemaform-process">
           <ol>
             <li className="process-step list-one">
-              <h3>Prepare</h3>
-              <p>To place an order, you'll need your:</p>
+              <h3 className="vads-u-font-size--h5">Prepare</h3>
+              <p>To place an order, you’ll need your:</p>
               <ul>
                 <li>Shipping address</li>
                 <li>Email address</li>
                 <li>Hearing aid information</li>
               </ul>
               <h4>What if I need help with my order?</h4>
               <p>
-                If you need help ordering hearing aid and betteries and
-                accessories, you cna contact the Devner Logistics Center (DLC)
-                at <a href="tel:303-273-6200">303-273-6200</a>
+                If you need help ordering hearing aid batteries and accessories,
+                you can call the Denver Logistics Center Customer Service
+                Section at
```

Comment:
Please add a `{' '}` here or `&nbsp;` because of this:

![Screen Shot 2020-04-09 at 4 09 48 PM](https://user-images.githubusercontent.com/136959/78941294-01ec8580-7a7d-11ea-827a-361b39216fe4.png)

Reply:
Done